plat/imx/common/imx_sip_svc.c

```diff
@@ -34,6 +34,11 @@ static uintptr_t imx_sip_handler(unsigned int smc_fid,
 		SMC_RET1(handle, imx_soc_info_handler(smc_fid, x1, x2, x3));
 		break;
 #endif
+#if defined(PLAT_imx8mm) || defined(PLAT_imx8mn) || defined(PLAT_imx8mp)
+	case IMX_SIP_SRC:
+		SMC_RET1(handle, imx_src_handler(smc_fid, x1, x2, x3, handle));
+		break;
+#endif
```

Comment:
NAK, this is already done below, see line 57 below.

Extend the conditional compilation with `PLAT_imx8mn` and `PLAT_imx8mp` instead.